plugins/ircd/irc2

```diff
@@ -70,8 +78,10 @@ my $irc = POE::Component::IRC->spawn(
    nick => $nickname,
    ircname => $ircname,
    server => $server,
+   port => $port,
    raw => 0,
-   useipv6 => 0,
+   usessl => $usessl,
```

Comment:
Are you sure, that the string (taken from `$ENV{USESSL}`) is handled properly? Or should you convert it into a numeric value? Is `USESSL=0` really interpreted as *no SSL*?

Reply:
Yes it's really interpreted as *no SSL* as [mentioned in the documentation](https://metacpan.org/pod/POE::Component::IRC#spawn), and maybe I can add and int conversion to be sure?